src/main/kotlin/uk/gov/digital/ho/hocs/queue/config/AwsConfiguration.kt

```diff
@@ -103,4 +107,22 @@ class AwsConfiguration(
       .withRegion(region)
       .withCredentials(AWSStaticCredentialsProvider(credentials)).build()
   }
+
+  @Bean(name = ["caseCreatorAwsSqsClient"])
```

Comment:
We don't actually need to SQS clients here as they both use the same access key and secret. 

Although it does beg the question whether they should...

Reply:
They do because Camel used to handle the DLQ and it was a limitation of that.